src/client/pages/_slug.vue

```diff
@@ -24,3 +30,51 @@ export default {
   computed: {},
 }
 </script>
+
+<style>
+.page {
+  margin-top: 80px;
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  grid-template-areas:
+    "header header"
```

Comment:
I would prefix all these names with `page-` so they don't conflict with grid names in other components.